src/openforms/submissions/tests/test_submission_status.py

```diff
@@ -383,6 +384,80 @@ def test_displaying_main_website_link_does_not_return_link_when_link_should_not_
             self.assertEqual(response_data["result"], ProcessingResults.success)
             self.assertEqual(response_data["mainWebsiteUrl"], "")
 
+    @patch(
+        "openforms.submissions.api.serializers.GlobalConfiguration.get_solo",
+        return_value=GlobalConfiguration(main_website="https://maykinmedia.nl"),
+    )
+    def test_displaying_main_website_link_from_theme_connected_to_form(
+        self, mock_get_solo
+    ):
+        theme = ThemeFactory.create(
+            organization_name="The company", main_website="http://example.com"
+        )
+        submission = SubmissionFactory.create(completed=True, form__theme=theme)
+
+        token = submission_status_token_generator.make_token(submission)
+        check_status_url = reverse(
+            "api:submission-status", kwargs={"uuid": submission.uuid, "token": token}
+        )
+
+        with patch("openforms.submissions.status.AsyncResult") as mock_AsyncResult:
+            mock_AsyncResult.return_value.state = states.SUCCESS
+            response = self.client.get(check_status_url)
+            response_data = response.json()
+
+            self.assertEqual(response_data["mainWebsiteUrl"], "http://example.com")
+
+    @patch(
+        "openforms.submissions.api.serializers.GlobalConfiguration.get_solo",
+        return_value=GlobalConfiguration(
+            default_theme=ThemeFactory.create(
+                organization_name="The company", main_website="http://example.com"
+            ),
+            main_website="https://maykinmedia.nl",
+        ),
+    )
```

Comment:
you get a DB error here because the `ThemeFactory.create` is evaluated when the test module is loaded/imported. That happens *before* Django has set up the test database (or the transaction for the test itself), and thus it tries to talk to the "real" database which doesn't exist.

Change this to `ThemeFactory.build` if possible, so you only have an in-memory object, or move the theme creation + return_value setup into the test method.